packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/address/move-eligibility-address.controller.js

```diff
@@ -134,20 +134,29 @@ export default class {
   }
 
   /**
-   * Copper eligibility by address
+   * Copper eligibility by address4
    */
   copperEligibilityByAddress() {
+    if (this.isReseller) {
+      // If current offer is a reseller offer,
+      // launch eligibility test address for partners (reseller)
+      return this.OvhApiConnectivityEligibility.v6()
+        .testAddressPartners(this.$scope, {
```

Comment:
The calls `testAddress` & `testAddressPartners` could be stored in a const, into the if, to reduce code duplication here.